server/src/db/index.ts

```diff
@@ -11,7 +30,7 @@ const db = pgPromise()({
 });
 
 const publicEventColumns = 'id, fancy_id, name, description, city, country, event_url, image_url, year, start_date, ' +
-  'end_date, event_host_id, from_admin, virtual_event'
+  'end_date, event_host_id, from_admin, virtual_event, event_template_id'
```

Comment:
Are we sure that altering this query does not changes the events API?

Reply:
Are you refering to the GET entpoint? it does not add event_template_id to GET endpoints if you dont add the field in response properties.

![Screenshot from 2020-07-30 13-19-34](https://user-images.githubusercontent.com/5005570/88947962-acc0c380-d267-11ea-83f8-139babe6a735.png)